cfgov/v1/util/ref.py

```diff
@@ -7,21 +7,21 @@
     ('document', 'Report'),
     ('date', 'Events'),
     ('microphone', 'Speech'),
-    ('bullhorn', 'Press Release'),
-    ('contract', 'Op-Ed'),
+    ('bullhorn', 'Press release'),
+    ('contract', 'Op-ed'),
     ('double-quote', 'Testimony'),
 ]
 
 related_posts_categories = [
     ('Blog', (
         ('At the CFPB', 'At the CFPB'),
-        ('Policy &amp; Compliance', 'Policy & Compliance'),
-        ('Data, Research &amp; Reports', 'Data, research & reports'),
+        ('Policy &amp; Compliance', 'Policy and compliance'),
```

Comment:
Note that we don't change the left side of the tuple here, because that's what's stored in the Wagtail block `specific_categories` for the page, and to update it we must also publish the page, which would cause other ripple effects, e.g. `date_published` on a bunch of pages would get updated and cause how we display recent posts to be out of order.

Reply:
That is... unfortunate.